shared/checks/oval/audit_rules_auditctl.xml

```diff
@@ -3,7 +3,7 @@
     <metadata>
       <title>Record Any Attempts to Run semanage</title>
       <affected family="unix">
-        <platform>Red Hat Enterprise Linux 7</platform>
+        <platform>multi_platform_rhel</platform>
```

Comment:
how could this OVAL possibly work on RHEL6? It relies on systemd!!

Reply:
Well, that was a huge oversight, :D

Reply:
PR #3056 should address this issue.

Reply:
Rebase this please

Reply:
Rebased.